### DISCUSSION TITLE
Transfer existing discussions to https://github.com/project-chip/matter.js/discussions ?

### DISCUSSION THREAD

**Discussion by Hedda** [mfucci/node-matter#276] [General]
@mfucci @Apollon77 if you have admin permissions for both repos on GitHub then can transfer [these discussions](https://github.com/mfucci/node-matter/discussions) to the [new repo](https://github.com/project-chip/matter.js/discussions). See:

https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository

https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#transferring-a-discussion

**Comment by Apollon77** [comment]
As said in the other thread: I think even mfucci do not have admin on the "target" project ... so no chance for that ...
